libr/util/print.c

```diff
@@ -935,7 +935,11 @@ R_API void r_print_hexdump(RPrint *p, ut64 addr, const ut8 *buf, int len, int ba
 					r_print_cursor (p, j, 1);
 					// stub for colors
 					if (p && p->colorfor) {
-						a = p->colorfor (p->user, n, true);
+						if (!p->iob.addr_is_mapped (p->iob.io, addr + j)) {
+							a = p->cons->pal.ai_unmap;
```

Comment:
_I'm still confused by this topic but..._ using p->cons instead of `p->consbind` should break dependencies again, but I've seen p->cons used a lot in print.c so I kept p->cons for consistency.